Resources/Prototypes/Entities/Mobs/NPCs/slimes.yml

```diff
@@ -0,0 +1,232 @@
+    damage:
+      types:
+        Asphyxiation: 0.2
+    damageRecovery:
+      types:
+        Asphyxiation: -1.0
+    maxSaturation: 15
+  - type: Damageable
+    damageContainer: Biological
+    damageModifierSet: Slime
+  - type: Bloodstream
+    bloodReagent: Slime
+    bloodlossDamage:
+      types:
+        Bloodloss:
+          1
+    bloodlossHealDamage:
+      types:
+        Bloodloss:
+          -0.25
+  - type: Barotrauma
+    damage:
+      types:
+        Blunt: 0.45
+  - type: Reactive
+    groups:
+      Flammable: [ Touch ]
+      Extinguish: [ Touch ]
+    reactions:
+    - reagents: [ Water, SpaceCleaner ]
+      methods: [ Touch ]
+      effects:
+      - !type:WashCreamPieReaction
+    - reagents: [ Water ]
+      methods: [ Touch ]
+      effects:
+      - !type:HealthChange
+        scaled: true
+        damage:
+          types:
+            Heat: 3
+      - !type:PopupMessage
+        type: Local
+        visualType: Large
+        messages: [ "slime-hurt-by-water-popup" ]
+        probability: 0.25
+  - type: CombatMode
+  - type: MeleeWeapon
+    hidden: true
+    soundHit:
+        path: /Audio/Weapons/punch3.ogg
+    angle: 0
+    animation: WeaponArcPunch
+    damage:
+      types:
+        Blunt: 8
+        Structural: 4
+        Caustic: 4
+  - type: InteractionPopup
+    successChance: 0.5
+    interactSuccessString: petting-success-slimes
+    interactFailureString: petting-failure-generic
+  - type: ReplacementAccent
+    accent: slimes
+  - type: GhostTakeoverAvailable
+    makeSentient: true
+    name: ghost-role-information-slimes-name
+    description: ghost-role-information-slimes-description
+
+- type: entity
+  name: blue slime
+  parent: MobAdultSlimes
+  id: MobAdultSlimesBlueAngry
+  suffix: Angry
+  components:
+    - type: Faction
+      factions:
+        - SimpleHostile
+
+- type: entity
+  name: green slime
+  parent: MobAdultSlimes
+  id: MobAdultSlimesGreenGood
+  suffix: Good
+  components:
+    - type: Sprite
+      layers:
+      - map: [ "enum.DamageStateVisualLayers.Base" ]
+        state: green_adult_slime
+    - type: DamageStateVisuals
+      states:
+        Alive:
+          Base: green_adult_slime
+        Dead:
+          Base: green_adult_slime_dead
+    - type: MeleeWeapon
+      damage:
+        types:
+          Blunt: 8
+          Structural: 4
+          Caustic: 1
+          Poison: 4
+
+- type: entity
+  name: green slime
+  parent: MobAdultSlimes
+  id: MobAdultSlimesGreenAngry
+  suffix: Angry
+  components:
+    - type: Faction
+      factions:
+        - SimpleHostile
+    - type: Sprite
+      layers:
+      - map: [ "enum.DamageStateVisualLayers.Base" ]
+        state: green_adult_slime
+    - type: DamageStateVisuals
+      states:
+        Alive:
+          Base: green_adult_slime
+        Dead:
+          Base: green_adult_slime_dead
+    - type: MeleeWeapon
+      damage:
+        types:
+          Blunt: 8
+          Structural: 4
+          Caustic: 1
```

Comment:
Isn't caustic hard to heal? Seems weird to just include 1 point of it.

Reply:
Reduce Blunt's damage to 6, and caustic is more of a tribute to being a slime, even if it's not his primary damage, it will always be at the slime.